src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -549,7 +549,7 @@ public bool IsNode(params RoleName[] nodes)
         /// </summary>
         public void EnterBarrier(params string[] name)
         {
-            TestConductor.Enter(RemainingOr(TestConductor.Settings.BarrierTimeout), name.ToImmutableList());
+            TestConductor.Enter(RemainingOr(TestConductor.Settings.BarrierTimeout), Myself, name.ToImmutableList());
```

Comment:
Whenever we call `EnterBarrier` during testing now, we always pass in `Myself` - which is set to the value of _our_ role during the testing system.